src/Devolutions.AvaloniaTheme.MacOS/Devolutions.AvaloniaTheme.MacOS.csproj

```diff
@@ -1,27 +1,26 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
-    <PropertyGroup>
-        <Version>1.0.0.0</Version>
-        <Company>Devolutions</Company>
-        <Description>Devolutions Avalonia macOS Theme</Description>
-        <PackageProjectUrl>https://github.com/Devolutions/avalonia-extensions</PackageProjectUrl>
-        <PackageLicenseExpression>MIT</PackageLicenseExpression>
-        <TargetFramework>net9.0</TargetFramework>
-        <ImplicitUsings>enable</ImplicitUsings>
-        <Nullable>enable</Nullable>
-    </PropertyGroup>
-
-    <ItemGroup>
-        <PackageReference Include="Avalonia" Version="[11.2.0,)"/>
-        <PackageReference Include="Avalonia.Controls.DataGrid" Version="[11.2.0,)"/>
-        <PackageReference Include="Avalonia.Svg.Skia" Version="[11.2.0,)"/>
-        <PackageReference Include="Avalonia.Themes.Fluent" Version="[11.2.0,)"/>
-    </ItemGroup>
-
-    <ItemGroup>
-        <AvaloniaResource Include="Accents/Assets/**"/>
-    </ItemGroup>
-
-</Project>
-
-
+  <PropertyGroup>
+    <Version>0.1.0-dev1</Version>
```

Comment:
What's the thought behind this version change? (I seem to remember querying the 1.0.0.0 at some point when it really felt more like 0.1. ... and Marc-André said it doesn't matter because by default our [nuget packaging script](https://github.com/Devolutions/avalonia-extensions/blob/master/.github/workflows/build-package.yml#L66) generates the version based on date anyway - but I'd still be a little wary of _down_-versioning here)

(Same on the Linux.csproj)

Reply:
That was generated by my local script that's building a local nuget for testing the theme in RDM, I accidently committed it but it shouldn't have been :sweat_smile: 

I'll clean that up

Reply:
whoops, still here ... or again 😆